packages/lsp11-contracts/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@lukso/lsp11-contracts",
+  "version": "0.0.1",
```

I guess?

```suggestion
  "version": "0.1.0",
```

Should it also be rc?

```suggestion
  "version": "0.1.0-rc.0",
```